changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 9.6.0 - xxxx-xx-xx =
+* Fix - Removes the subscription detached notice feature to avoid issues when requesting the Stripe's API to retrieve payment method details.
```

Comment:
This changelog entry is a bit confusing to me. Maybe we can phrase it as follows?
```suggestion
* Fix - Stop checking for detached subscriptions for admin users, as it was slowing down wp-admin
```